docs/contributor/04-20-release.md

```diff
@@ -1 +1,51 @@
 # Kyma Environment Broker release pipeline
+
+## Overview
+
+The Kyma Environment Broker release pipeline creates proper artifacts:
+ - kyma-environment-broker Docker image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/kyma-environment-broker)
+ - kyma-environment-deprovision-retrigger Docker image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/kyma-environment-deprovision-retrigger)
+ - kyma-environments-cleanup-job Docker image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/kyma-environments-cleanup-job )
+ - kyma-environment-runtime-reconciler Docker image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/kyma-environment-runtime-reconciler)
+ - kyma-environment-trial-cleanup-job Docker image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/kyma-environment-trial-cleanup-job)
+ - kyma-environment-subaccount-cleanup-job Docker image in the [registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/kyma-environment-subaccount-cleanup-job)
+
+## Run the pipeline
+
+### Create a release
+
+![Release diagram](../assets/release.svg)
+
+To create a release, follow these steps:
+
+1. Run GitHub action **Create release**:  
+   i.  go to the **Actions** tab  
+   ii. click on **Create release** workflow   
+   iii. click  **Run workflow** on the right  
+   iv. provide a version, for example, 1.2.0  
+   v. choose whether to bump or not to bump the security scanner config  
+   vi. enter a number of last k3s versions to be used for KEB chart tests or leave empty (default value is 3)
+   
+2. The GitHub action, defined in the [`create-release.yaml`](/.github/workflows/create-release.yaml) file, validates the release by checking if the GitHub tag already exists, if there are any old Docker images for that GitHub tag, and if merged PRs that will be part of this release are labeled correctly.
+3. The GitHub action asynchronously initiates unit tests, KEB chart tests, and if you chose in step 1.v to bump the security scanner config, the GitHub action creates a PR with a new security scanner config that includes the new GitHub tag version.
```

Comment:
```suggestion
3. The GitHub action asynchronously initiates unit tests and KEB chart tests. If, in step 1.v, you chose to bump the security scanner config, the GitHub action creates a PR with a new security scanner config that includes the new GitHub tag version.
```

Reply:
The bump is also asynchronous if the users choose to do it, so I'm not sure if we can change `,` to `and` as it may look like only unit tests and KEB chart test are run asynchronously.